### PR TITLE
Make use of actor middleware

### PR DESCRIPTION
**Pull request by thbaan**

### CODE REVIEW

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/InternalCommands/QueuedInternalCommand.cs [hunk 1]
```diff
@@ -48,6 +49,8 @@ public QueuedInternalCommand(Guid id, string type, byte[] data, Instant creation
 
         public string CorrelationId { get; }
 
+        public string CurrentActor { get; }
```
[the remark this answers is not in the document]
Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/InternalCommands/QueuedInternalCommand.cs]: Fixed.

**Review by cmgrd — APPROVED** [review]

File: source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Model/202201180945 Add current actor to internal queue commands.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+ALTER TABLE QueuedInternalCommands
+    ADD CurrentActor NVARCHAR(MAX) NULL
```
Comment (Korgath) [inline source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Model/202201180945 Add current actor to internal queue commands.sql:2]: Is this necessary if you are dropping it immediately after?

**Review by Korgath — CHANGES_REQUESTED** [review]

File: source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Model/202201180945 Add current actor to internal queue commands.sql (hunk 2 above)
Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Model/202201180945 Add current actor to internal queue commands.sql]: Removed

**Review by Korgath — DISMISSED** [review]

File: source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Seed/202201181200 Fix actor ids.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO [dbo].[Actor] ([Id], [IdentificationNumber], [IdentificationType], [Roles])
```
Comment (MrDach) [inline source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Seed/202201181200 Fix actor ids.sql:1]: Are these scripts still relevant after we've implemented the "proper" actor registry?

**Review by MrDach — CHANGES_REQUESTED** [review]

**Review by MrDach — APPROVED** [review]

File: source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Seed/202201181200 Fix actor ids.sql (hunk 3 above)
Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/Scripts/Seed/202201181200 Fix actor ids.sql]: Reverted.

**Review by cmgrd — APPROVED** [review]

**Review by MrDach — APPROVED** [review]

**Pull request merged**